[qtwebengine] configuration failiure on x64-linux-dynamic

### Operating system

Linux

### Compiler

GCC 13.4.0

### Steps to reproduce the behavior

```Shell
`./vcpkg install --triplet=x64-linux-dynamic --host-triplet=x64-linux-dynamic qtwebengine`
```

### Failure logs

```
-- Configuring x64-linux-dynamic-dbg                                                                                                                                                          
-- Configuring x64-linux-dynamic-rel                                                                                                                                                          
CMake Warning at vcpkg_installed/x64-linux-dynamic/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:344 (message):                                                                               
  The following variables are not used in CMakeLists.txt:                                      
                                                                                               
      FEATURE_webengine_system_protobuf                                                        
                                                                                                                                                                                              
  Please recheck them and remove the unnecessary options from the                                                                                                                             
  `vcpkg_cmake_configure` call.                                                                
                                                                                                                                                                                              
  If these options should still be passed for whatever reason, please use the                                                                                                                 
  `MAYBE_UNUSED_VARIABLES` argument.                                                           
Call Stack (most recent call first):                                                                                                                                                          
  vcpkg_installed/x64-linux-dynamic/share/qtbase/qt_install_submodule.cmake:158 (vcpkg_cmake_configure)                                                                                       
  custom-ports/qtwebengine/portfile.cmake:215 (qt_cmake_configure)                                                                                                                            
  scripts/ports.cmake:206 (include)                                                                                                                                                           
                                                                                                                                                                                              
                                                                                                                                                                                              
CMake Error at custom-ports/qtwebengine/portfile.cmake:240 (file):                                                                                                                              file failed to open for writing (Is a directory):                                                                                                                                           
                                                                                                                                                                                              
    /root/vcpkg/buildtrees/qtwebengine/x64-linux-dynamic-dbg/src/core/Debug/                                                                                                                     
Call Stack (most recent call first):                                                                                                                                                          
  scripts/ports.cmake:206 (include)  
```

### Additional context

_No response_

@dg0yt 

```
Installing 19/19 qtwebengine[core,pdf,webengine]:x64-linux-dynamic@6.9.1...                                                                                                                   
Building qtwebengine[core,pdf,webengine]:x64-linux-dynamic@6.9.1...                                                                                                                           
/root/vp/triplets/community/x64-linux-dynamic.cmake: info: loaded community triplet from here. Community triplets are not built in the curated registry and are thus less likely to succeed.
-- Using cached html5lib-1.1-py2.py3-none-any.whl
-- Using cached six-1.17.0-py2.py3-none-any.whl
-- Using cached webencodings-0.5.1-py2.py3-none-any.whl
-- Setting up python virtual environment...
-- Installing python packages: --no-index;/root/vp/downloads/html5lib-1.1-py2.py3-none-any.whl;/root/vp/downloads/six-1.17.0-py2.py3-none-any.whl;/root/vp/downloads/webencodings-0.5.1-py2.py3-none-any.whl
-- Setting up python virtual environment... finished.
-- Using cached qtwebengine-everywhere-src-6.9.1.tar.xz
-- Cleaning sources at /root/vp/buildtrees/qtwebengine/src/here-src-6-228bf7d69e.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /root/vp/downloads/qtwebengine-everywhere-src-6.9.1.tar.xz
-- Applying patch clang-cl.patch
-- Applying patch cross-build.diff
-- Applying patch disable-host-pkgconfig.diff
-- Applying patch fix-error2275-2672.patch
-- Applying patch nested-name-fix.patch
-- Applying patch osx-sdk-info.diff
-- Applying patch pdf-system-libjpeg.diff
-- Applying patch pdf-system-libpng.diff
-- Applying patch pkg-config.diff
-- Applying patch rpath.diff
-- Using source at /root/vp/buildtrees/qtwebengine/src/here-src-6-228bf7d69e.clean
-- Configuring x64-linux-dynamic-dbg
-- Configuring x64-linux-dynamic-rel
CMake Error at buildtrees/versioning_/versions/qtwebengine/20af3e13a13986979309806ed3db5dd9f6926757/portfile.cmake:234 (file):
  file failed to open for writing (Is a directory):

    /root/vp/buildtrees/qtwebengine/x64-linux-dynamic-dbg/src/core/Debug/
Call Stack (most recent call first):
  scripts/ports.cmake:206 (include)


error: building qtwebengine:x64-linux-dynamic failed with: BUILD_FAILED
See https://learn.microsoft.com/vcpkg/troubleshoot/build-failures?WT.mc_id=vcpkg_inproduct_cli for more information.
Elapsed time to handle qtwebengine:x64-linux-dynamic: 3 min
```

Also:

```
bash-4.4# ls buildtrees/qtwebengine/x64-linux-dynamic-dbg/src/core/
CMakeFiles  CTestTestfile.cmake  Debug  api  cmake_install.cmake  host_toolchain  target_toolchain  tools  v8_toolchain
bash-4.4# ls buildtrees/qtwebengine/x64-linux-dynamic-rel/src/core/
CMakeFiles  CTestTestfile.cmake  Release  api  cmake_install.cmake  host_toolchain  target_toolchain  tools  v8_toolchain
```

--- a/ports/qtwebengine/portfile.cmake
+++ b/ports/qtwebengine/portfile.cmake
@@ -226,7 +226,7 @@ qt_cmake_configure(
 )
 
 vcpkg_backup_env_variables(VARS PKG_CONFIG_PATH)
-file(GLOB target_args_gn RELATIVE "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-dbg/src/core/Debug" "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-dbg/src/core/Debug/*/args.gn")
+file(GLOB target_args_gn RELATIVE "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel/src/core/Release" "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel/src/core/Release/*/args.gn")
 if(NOT VCPKG_BUILD_TYPE)
     block(SCOPE_FOR VARIABLES)
     set(VCPKG_BUILD_TYPE debug)

--- a/ports/qtwebengine/portfile.cmake
+++ b/ports/qtwebengine/portfile.cmake
@@ -226,7 +226,7 @@ qt_cmake_configure(
 )
 
 vcpkg_backup_env_variables(VARS PKG_CONFIG_PATH)
-file(GLOB target_args_gn RELATIVE "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-dbg/src/core/Release" "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-dbg/src/core/Release/*/args.gn")
+file(GLOB target_args_gn RELATIVE "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-dbg/src/core/Debug" "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-dbg/src/core/Debug/*/args.gn")
 if(NOT VCPKG_BUILD_TYPE)
     block(SCOPE_FOR VARIABLES)
     set(VCPKG_BUILD_TYPE debug)

--- a/ports/qtwebengine/vcpkg.json
+++ b/ports/qtwebengine/vcpkg.json
@@ -2,6 +2,7 @@
   "$comment": "x86-windows is not within the upstream support matrix of Qt6",
   "name": "qtwebengine",
   "version": "6.9.1",
+  "port-version": 1,
   "description": "Qt WebEngine provides functionality for rendering regions of dynamic web content.",
   "homepage": "https://www.qt.io/",
   "license": null,

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -8146,7 +8146,7 @@
     },
     "qtwebengine": {
       "baseline": "6.9.1",
-      "port-version": 0
+      "port-version": 1
     },
     "qtwebsockets": {
       "baseline": "6.9.1",

--- a/versions/q-/qtwebengine.json
+++ b/versions/q-/qtwebengine.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "70c6f26e30f66b96d234231ac17f2d46d0753d07",
+      "git-tree": "45f1164b10dc967d15b3b956834f57e319d7b34e",
       "version": "6.9.1",
       "port-version": 1
     },

--- a/versions/q-/qtwebengine.json
+++ b/versions/q-/qtwebengine.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "70c6f26e30f66b96d234231ac17f2d46d0753d07",
+      "version": "6.9.1",
+      "port-version": 1
+    },
+    {
       "git-tree": "20af3e13a13986979309806ed3db5dd9f6926757",
       "version": "6.9.1",
       "port-version": 0

